index.ja.md

```diff
@@ -0,0 +1,17 @@
+*2021-03-23*
+
+RMS として良く知られるリチャード M. ストールマンは長い間フリーソフトウェアコミュニティの危険な実力者であり続けてきました。彼は不当かつ深刻な嫌悪、とりわけ女性嫌悪主義、障害者差別主義、トランスジェンダー嫌悪主義を示してきました。フリーソフトウェア、ディジタル権、テック・コミュニティの中にこのような信条の居場所はありません。フリーソフトウェア財団 (FSF) 理事会への彼の復帰に際して、我々は FSF 理事会全員の退任および RMS がすべての指導的地位から解かれることを求めます。
+
+我々署名者一同は、ディジタル自治の必要性と基本的人権を保護する上でユーザの自由が果たす強力な役割を信じています。ソフトウェアの自由が可能にするすべての約束を実現するために、コミュニティ内に抜本的な変革が必要なのです。我々はすべてのテクノロジーが人々に抑圧でなく、力を与える現在と未来を支持します。これはテクノロジーがその最も基礎的な水準から、我々の権利を尊重して構築される世界でのみ可能なことです。リチャード M. ストールマンはこれらの理念を種々の形で広めましたが、我々のために語ったのではありませんでした。我々は彼の行動や言動を見過ごさないし、彼の指導や今日のフリーソフトウェア財団の指導を承認しません。
+
+RMS の言行不一致に耐えるのはもう沢山です。我々は一人の人間が我々の仕事の意味を台無しにさせるままにすることはできません。我々のコミュニティに RMS のような人物の居場所はなく、もう彼の行為に苦しめられたり、指導的役割を与えたり、彼自身や彼の有害かつ危険なイデオロギーを許容しつづけることはありません。
```

Comment:
「もう沢山です」は少々感情的な印象を与えはしないかと危惧しております。
「RMS の言行不一致に対して、人々は寛容にふるまってきました。しかしもはや我々は……」とかいかがでしょう。

(I point out here that his/her translation sounds to me like "We've had enough of his repugnant ideas and behavior!", but it might just for me.)